### PR TITLE
Fix simple test for Cascade Mask R-CNN

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
what about test and expected output? The test has been commented, you just need to uncomment and add expected output

**Comment by druzhkov-paul** [comment]
What was the purpose of #11 then?

**Comment by Ilya-Krylov** [comment]
#11 is like a reminder that we have some problems. Tests have been commented to make invisible (for a while) CI green. Now since you have been fixed the cascade mask rcnn issue, you can uncomment the test add expected output. I assume that CI should be green. As soon as your PR is merged I rebase #11 and there will be only 3 uncommented tests instead of 4 .

How does it sound to you?

**Comment by druzhkov-paul** [comment]
It sounds like you should've been more explicit in your intentions. :smile:
Ok, get the idea now. Will uncomment/add tests, no worries.

File: tools/convert_to_ir.py [hunk 1]
```diff
@@ -1,63 +0,0 @@
-# Copyright (C) 2020 Intel Corporation
```
Comment (Ilya-Krylov) [inline tools/convert_to_ir.py]: mr RewriteEverythingEachTime please update documentation as well

Comment (druzhkov-paul) [reply tools/convert_to_ir.py]: This commit should not be in this PR. Sorry for that.

File: tests/test_forward.py [hunk 2]
```diff
@@ -168,57 +168,57 @@ def test_retina_ghm_forward():
...
-#         img_metas,
-#         gt_bboxes=gt_bboxes,
-#         gt_labels=gt_labels,
-#         return_loss=True)
-#     assert isinstance(losses, dict)
-#     from mmdet.apis.train import parse_losses
-#     total_loss = float(parse_losses(losses)[0].item())
-#     assert total_loss > 0
+def test_cascade_forward():
+    try:
```
Comment (Ilya-Krylov) [inline tests/test_forward.py]: could you remove this try..except block in order not to skip test
please to it in other skipped tests.

looks like it does not work because of different torchvision

locally I successfully run two other tests

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by AlexanderDokuchaev** [comment]
run tests

**Review by Ilya-Krylov — APPROVED** [review]
